seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1495,8 +1519,14 @@ def _add_file_iter(self, stdout):
         self.mock_file_iter.return_value += stdout
 
 
-class AnvilDataManagerAPITest(AnvilAuthenticationTestCase, DataManagerAPITest):
-    fixtures = ['users', '1kg_project', 'reference_data']
+@mock.patch('seqr.views.utils.permissions_utils.PM_USER_GROUP', 'project-managers')
+class AnvilDataManagerAPITest(AirflowTestCase, DataManagerAPITest):
```

Comment:
Previously we ran `test_load_data` in its own `AirflowTestCase` and everything else in this class, which inherits from `AirtableTest`. Since loading data now accesses airflow and airtable, this merges those classes into one to take advantage of both utilities